stabilizer/src/main/java/com/hazelcast/stabilizer/agent/workerjvm/WorkerJvmLauncher.java

```diff
@@ -124,10 +130,27 @@ private WorkerJvm startWorkerJvm(String mode) throws IOException {
         new WorkerJvmProcessOutputGobbler(process.getInputStream(), new FileOutputStream(logFile)).start();
         workerJvm.process = process;
         workerJvm.mode = WorkerJvm.Mode.valueOf(mode.toUpperCase());
+        copyResourcesToWorkerId(workerId);
         workerJVMs.put(workerId, workerJvm);
         return workerJvm;
     }
 
+    private void copyResourcesToWorkerId(String workerId) throws IOException {
+        final String testSuiteId = agent.getTestSuite().id;
+        if (!new File(WORKERS_PATH + "/" + testSuiteId + "/upload/").exists()) {
+            log.debug("Skipping copy, since no copy file at the agent");
```

Comment:
Very confusing message. "Skip copying upload directory to workers since no upload directory was found" e.g.